lms/djangoapps/teams/views.py

```diff
@@ -1087,3 +1116,38 @@ def delete(self, request, team_id, username):
             return Response(status=status.HTTP_204_NO_CONTENT)
         else:
             return Response(status=status.HTTP_404_NOT_FOUND)
+
+
+class SearchResultPaginator(object):
```

Comment:
I would say either put it in a file called "paginators" (like "serializers"), or just keep it as-is. No need to make it private within TeamsListView-- that is apparently "un-pythonic".